src/Pages/ErrorHandlind.jsx

```diff
@@ -0,0 +1,7 @@
+export default function ErrorHandling() {
```

Comment:
Small one - i'd maybe just name this component Display 404, or something like that, if its only going to be used for this.